config/default.json

```diff
@@ -1,8 +1,14 @@
 {
-     "dbConfig": {
+    "dbConfig": {
         "host": "localhost"
     },
     "settings": {
         "adminAccounts": "fakepassword"
+    },
+    "recipient" : {
+        "to" : ["aiaimforworld@gmail.com",  "benevbright@gmail.com"]
+    },
+    "sender" : {
+        "from" : "admin@catsdogssoftball.com"
```

Comment:
Let's name it `no_reply@catsdogssoftball.com`. 
We would want to use it for the new players too. So they can know it's supposed not to reply.